src/components/RightSidebar/Participants/Participant.vue

```diff
@@ -864,22 +862,20 @@ export default {
 			}
 			return ''
 		},
-	},
-	watch: {
-		isParticipantSpeaking(speaking) {
-			if (speaking) {
-				if (!this.speakingInterval) {
-					this.speakingInterval = setInterval(this.computeElapsedTime, 1000)
-				}
-			} else {
-				if (speaking === undefined) {
-					this.timeSpeaking = 0
-				}
-				clearInterval(this.speakingInterval)
-				this.speakingInterval = null
+
+		timeSpeaking() {
+			if (!this.participantSpeakingInformation) {
+				return null
+			}
+			if (this.isParticipantSpeaking === undefined) {
+				return 0
 			}
```

Comment:
```suggestion
			if (!this.participantSpeakingInformation || !this.isParticipantSpeaking) {
				return null
```

No need to differentiate, boolean value of `timeSpeaking` will be the same

Reply:
formattedTime expects `number`, and `this..isParticipantSpeking` could be `false`. But good catch anyway